modules/tinymce/src/themes/silver/main/ts/ui/dialog/ColorInput.ts

```diff
@@ -190,8 +190,8 @@ export const renderColorInput = (spec: ColorInputSpec, sharedBackstage: UiFactor
             Composing.getCurrent(comp).each(Focusing.focus);
           });
         }),
-        AlloyEvents.run<ColorPickerCancelEvent>(colorPickerCancelEvent, (comp, se) => {
-          FormField.getField(comp).each((field) => {
+        AlloyEvents.run<ColorPickerCancelEvent>(colorPickerCancelEvent, (comp, _se) => {
+          FormField.getField(comp).each((_field) => {
```

Comment:
This also looks to do nothing? So we could probably just do `Composing.getCurrent(comp).each(Focusing.focus);` directly.

Reply:
In this case I think the `FormField.getField(comp).each((_field) => {` was testing if there were any form fields before requesting focus.